indexes/core_guidelines/types.ts

```diff
@@ -0,0 +1,8 @@
+export type core_guidelines_entry = {
+    id: string;
+    caption: string;
+    anchor: string;
+    title: string;
+}
+
+export type core_guidelines_index = core_guidelines_entry[];
```

Comment:
Super minor but can we get a newline at the end of this file

Reply:
Absolutely, my bad.